### ISSUE TITLE
nf-core tools `bump-version` command does not preserve formatting

### ISSUE THREAD

**Issue by fabianegli** [nf-core/tools#2206]
### Description of feature

When running `nf-core bump-version 1.1.0` I get the following diff:

```
% git diff
diff --git a/nextflow.config b/nextflow.config
index ee0fca4..c249288 100644
--- a/nextflow.config
+++ b/nextflow.config
@@ -366,7 +366,7 @@ manifest {
     description     = """Quantitative Mass Spectrometry nf-core workflow"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=22.10.1'
-    version         = '1.1dev'
+    version = '1.1.0'
     doi             = ''
 }
```

but would expect

```
% git diff
diff --git a/nextflow.config b/nextflow.config
index ee0fca4..c249288 100644
--- a/nextflow.config
+++ b/nextflow.config
@@ -366,7 +366,7 @@ manifest {
     description     = """Quantitative Mass Spectrometry nf-core workflow"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=22.10.1'
-    version         = '1.1dev'
+    version         = '1.1.0'
     doi             = ''
 }
```
instead.

**Comment by fabianegli** [comment]
Sorry for the dupe 🤭 @maxulysse

**Comment by maxulysse** [comment]
> Sorry for the dupe 🤭 @maxulysse

no worries, it happens to the best ;-)

**Comment by fabianegli** [comment]
Closing as fixed in #2514
